### ISSUE TITLE
onCreateMenu

### ISSUE THREAD

**Issue by amirnourazar** [mariohmol/ang-jsoneditor#55]
Hi,

I need your help in implementing "OnCreateMenu".
it seems "OnCreateMenu" option is not available in ang-jsoneditor, so i need your help in implementing it by my self.
where should i begin?

**Comment by mariohmol** [comment]
Hi!
Thanks for that.. I think you have to make something like this

https://github.com/mariohmol/ang-jsoneditor/blob/master/ang-jsoneditor/src/jsoneditor/jsoneditor.component.ts#L128

**Comment by amirnourazar** [comment]
thanks, 
I tried to build the project but I got an error:
[error] TypeError: Object prototype may only be an Object or null: undefined
    at setPrototypeOf (<anonymous>)
    at Object.__extends (D:\Projects\ang-jsoneditor-master\node_modules\tslib\tslib.js:65:9)
    at D:\Projects\ang-jsoneditor-master\node_modules\@angular\compiler-cli\src\ngtsc\indexer\src\template.js:115:17
    at D:\Projects\ang-jsoneditor-master\node_modules\@angular\compiler-cli\src\ngtsc\indexer\src\template.js:314:6
    at D:\Projects\ang-jsoneditor-master\node_modules\@angular\compiler-cli\src\ngtsc\indexer\src\template.js:3:17
    at Object.<anonymous> (D:\Projects\ang-jsoneditor-master\node_modules\@angular\compiler-cli\src\ngtsc\indexer\src\template.js:9:3)
    at Module._compile (internal/modules/cjs/loader.js:868:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:879:10)
    at Module.load (internal/modules/cjs/loader.js:731:32)
    at Function.Module._load (internal/modules/cjs/loader.js:644:12)
    at Module.require (internal/modules/cjs/loader.js:771:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at D:\Projects\ang-jsoneditor-master\node_modules\@angular\compiler-cli\src\ngtsc\indexer\src\transform.js:20:22
    at D:\Projects\ang-jsoneditor-master\node_modules\@angular\compiler-cli\src\ngtsc\indexer\src\transform.js:10:17
    at Object.<anonymous> (D:\Projects\ang-jsoneditor-master\node_modules\@angular\compiler-cli\src\ngtsc\indexer\src\transform.js:16:3)
    at Module._compile (internal/modules/cjs/loader.js:868:30)

**Comment by mariohmol** [comment]
isnt the typescript version?

check my env


```
ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 8.1.2
Node: 12.2.0
OS: darwin x64
Angular: 8.1.3
... common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.6
@angular-devkit/build-angular     0.800.6
@angular-devkit/build-optimizer   0.800.6
@angular-devkit/build-webpack     0.800.6
@angular-devkit/core              8.0.6
@angular-devkit/schematics        0.6.8
@angular/cli                      8.1.2
@angular/http                     7.2.15
@ngtools/json-schema              1.1.0
@ngtools/webpack                  8.0.6
@schematics/angular               8.1.2
@schematics/update                0.801.2
ng-packagr                        3.0.6
rxjs                              6.5.2
typescript                        3.4.3
webpack                           4.38.0
```

**Comment by mariohmol** [comment]
i;m getting this error here as well https://travis-ci.org/mariohmol/ang-jsoneditor/builds/601796867

**Comment by mariohmol** [comment]
included in 1.9.1 , see if that works for u

**Comment by MichaelPanwenjie** [comment]
In 1.9.1 , it still not found.
Which version should i get?
